### DISCUSSION TITLE
how to print variables and internal extractors name values in cli results

### DISCUSSION THREAD

**Discussion by passwa11** [projectdiscovery/nuclei#4215] [Q&A]
without use -o  -debnug  or -proxy ,get the variables and internal extractors name values in cli results

**Comment by MetzinAround** [comment]
Not sure if you've seen our docs page on using extractors, but this might be a helpful place to start. Let us know fi you don't find waht you're looking for there. https://docs.nuclei.sh/template-guide/operators/extractors#xpath-extractor

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
